BrainPortal/app/views/userfiles/_default_qc_panel.html.erb

```diff
@@ -22,15 +22,29 @@
 #
 -%>
 
-<% if ! @userfile.is_locally_synced? %>
-  This file must be synced locally to view QC data.
-<% else %>
-  <% if @userfile.is_a?(ImageFile) %>
-    <%= image_tag url_for(:action  => :content, :id => @userfile.id) %><br>
+<% div_id_to_replace = "sub_viewer_#{@userfile.id}" %>
+
+<div id=<%= div_id_to_replace %>>
+  <% if ! @userfile.is_locally_synced? %>
+    This file must be synced locally to view QC data.
   <% else %>
-    <span class="warning">
-    This file does not seem to contain any QC data, or no template is available to QC this file type.
-    </span>
+    <% link = data_link(@userfile.name, @userfile, div_id_to_replace) %>
+    <% if link =~ /data-url=/%>
+        <%= link %>
+      </div>
+    <% else %>
+      <span class="warning">
+        This file does not seem to contain any QC data, or no template is available to QC this file type.
+      </span>
+    <% end %>
   <% end %>
-<% end %>
+</div>
+
+<script type="application/javascript">
+  // Find the link inside the div and click it
+  var subViewerDiv  = $("#<%= div_id_to_replace %>");
+  var subViewerLink = subViewerDiv.find(".ajax_onclick_show_element");
```

Comment:
What is this class? How is it added to the HTML elements? If it's done later after the page is loaded, could this script be executed too early? Maybe the script tag shoudl use the attribute `defer` ?

Reply:
Try this in a bran new browser in incognito and no cached javascript

Reply:
Done, it works fine in an incognito browser.